### PR TITLE
Create addresses_events for mantle

### PR DESCRIPTION
**Pull request by hildobby**
### Contribution type
Please check the type of contribution this pull request is for:

- [ ] New spell(s)
- [ ] Adding to existing spell lineage
- [ ] Bug fix

**Note:** You can safely discard any section below which doesn't apply based on selection above

---

### For new spell(s)
If you are building new spell(s), please provide the following information:

- **Spell name(s):** *schema.table_name*
- **Description:** [Detailed description of the new spell(s) and their purpose]
- **Who are the new spell(s) for?** [Internal team or general community]
- **How will the new spell(s) be used downstream?** [Description of downstream usage]
- **Implementation details:** [Information on how the spell(s) are implemented]
- **Test instructions:** [How to test the new spell(s)]
- **Related issue(s):** [Link to related issues, if any]

---

### For adding to existing spell lineage
If you are adding to an existing spell lineage, please provide the following information:

- **Description:** [Description of the changes made]

---

### For bug fixes
If you are fixing a bug, please provide the following information:

- **Description:** [Description of the bug fix]
- **Steps to reproduce:** [How to reproduce the bug]
- **Implementation details:** [Information on how the bug was fixed]
- **Test instructions:** [How to test the fix]
- **Related issue(s):** [Link to related issues, if any]

---

### Additional information
Please provide any additional information that might help us review your pull request:

- [Any additional information]

---

Thank you for your contribution!

### CODE REVIEW

File: daily_spellbook/models/addresses_events/mantle/addresses_events_mantle_first_activity.sql [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    FROM
+    {{ source('abi', 'signatures') }}
+    where type = 'function_call'
+    AND id NOT IN (0x09779838, 0x00000000) -- for some weird reason these have duplicates functions
+) sig
+    ON sig.id = bytearray_substring(et.data, 1, 4)
+{% if is_incremental() %}
+LEFT JOIN {{this}} ffb ON et."from" = ffb.address
+WHERE ffb.address IS NULL
+AND et.block_time >= date_trunc('day', now() - interval '7' day)
```
Comment (Hosuke) [inline daily_spellbook/models/addresses_events/mantle/addresses_events_mantle_first_activity.sql:34]:
```suggestion
AND {{ incremental_predicate('et.block_time') }}
```

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @hildobby 

[Sample addresses_events_mantle_first_funded_by]()
[Sample addresses_events_mantle_first_activity]()

**Review by jeff-dude — APPROVED** [review]
@Hosuke i'm going to revert the fix on incremental predicate for now. these spells exist for a lot of other chains that all need refactored together. figured it'd be easier to track to have them all in the same state when doing so.

File: daily_spellbook/models/addresses_events/mantle/addresses_events_mantle_schema.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      blockchain: mantle
+      sector: addresses_events
+      project: addresses
+      contributors: hildobby
+    config:
+      tags: ['table', 'funded', 'addresses_events', 'mantle']
+    description: "Table showing who first funded each Linea address in ETH"
+    tests:
+      - check_seed:
+          seed_file: ref('first_funded_by_seed')
```
[the remark this answers is not in the document]
Comment (jeff-dude) [reply daily_spellbook/models/addresses_events/mantle/addresses_events_mantle_schema.yml:15]: i think it's okay on some of these spells that fall into the daily project to not have the seeds. similar to incremental predicate below, we will want to clean out schema file when changing.

**Pull request merged**